### ISSUE TITLE
Not working with Yarn berry

### ISSUE THREAD

**Issue by devunt** [HoudiniGraphql/houdini#1035]
### Describe the bug

When using Yarn's PnP mode, houdini vite plugin cannot locate `houdini-svelte` package and refuses to run.
After looking the code, it seems it tries to locate the package by looking into `node_modules` directory, which does not exist in Yarn PnP mode.

```
error when starting dev server:
Error: Could not find plugin: houdini-svelte. Are you sure its installed? If so, please open a ticket on GitHub.
    at pluginPath (file:///Users/devunt/workspace/rezero/.yarn/unplugged/houdini-npm-1.1.5-364ec1b613/node_modules/houdini/build/vite-esm/index.js:69859:17)
    at getConfig (file:///Users/devunt/workspace/rezero/.yarn/unplugged/houdini-npm-1.1.5-364ec1b613/node_modules/houdini/build/vite-esm/index.js:69730:28)
    at async config (file:///Users/devunt/workspace/rezero/.yarn/unplugged/houdini-npm-1.1.5-364ec1b613/node_modules/houdini/build/vite-esm/index.js:75905:17)
    at async runConfigHook (file:///Users/devunt/workspace/rezero/.yarn/__virtual__/vite-virtual-60255e3d38/0/cache/vite-npm-4.2.1-a0fa6996de-70eb162ffc.zip/node_modules/vite/dist/node/chunks/dep-79892de8.js:64005:25)
    at async resolveConfig (file:///Users/devunt/workspace/rezero/.yarn/__virtual__/vite-virtual-60255e3d38/0/cache/vite-npm-4.2.1-a0fa6996de-70eb162ffc.zip/node_modules/vite/dist/node/chunks/dep-79892de8.js:63499:14)
    at async createServer (file:///Users/devunt/workspace/rezero/.yarn/__virtual__/vite-virtual-60255e3d38/0/cache/vite-npm-4.2.1-a0fa6996de-70eb162ffc.zip/node_modules/vite/dist/node/chunks/dep-79892de8.js:62752:20)
    at async CAC.<anonymous> (file:///Users/devunt/workspace/rezero/.yarn/__virtual__/vite-virtual-60255e3d38/0/cache/vite-npm-4.2.1-a0fa6996de-70eb162ffc.zip/node_modules/vite/dist/node/cli.js:734:24)
```

### Severity

serious, but I can work around it

### Steps to Reproduce the Bug

Try to use with Yarn 3

**Comment by AlecAivazis** [comment]
Hmm... can you think of a way for us to locate the filepath for the installed module when pnp is enabled? Do we need to somehow detect if its running in pnp mode to look elsewhere or is there a universal way we can look that up?

**Comment by devunt** [comment]
> Hmm... can you think of a way for us to locate the filepath for the installed module when pnp is enabled? Do we need to somehow detect if its running in pnp mode to look elsewhere or is there a universal way we can look that up?

Unfortunately we have to handle the PnP environment separately and it is possible by checking `process.versions.pnp` property. I have implemented the patch #1037.
